src/main/resources/templates/db_scripts/schema.json

```diff
@@ -208,7 +208,9 @@
       "foreignKeys": [
         {
           "fieldName": "organizationId",
-          "targetTable": "organizations"
+          "targetTable": "organizations",
+          "tableAlias": "bankingInformation",
+          "targetTableAlias": "organizations"
```

Comment:
better to use singular form

Reply:
![image](https://github.com/folio-org/mod-organizations-storage/assets/113523904/3f83a61b-4754-425e-84f9-d1e814b52923)
it looks like plural form is common practise. can we keep it as it was?